kolibri/plugins/user/assets/src/views/sign-in-page/index.vue

```diff
@@ -288,8 +311,8 @@
           this.kolibriLogin({
             username: this.username,
             password: this.password,
-            facility: this.facility,
-          });
+            facility: this.facilityConfig.id,
```

Comment:
Why is this set to the id of the facilityConfig object? This is not a facility, it's a facilitydataset's id.

Reply:
That was a mistake ^ found this out the hard way